src/opendr/perception/object_detection_2d/yolov3/yolov3_learner.py

```diff
@@ -572,21 +572,38 @@ def download(self, path=None, mode="pretrained", verbose=False,
                                     "yolo_voc.json")
             if verbose:
                 print("Downloading metadata...")
-            urlretrieve(file_url, os.path.join(path, "yolo_default.json"))
+            if not os.path.exists(os.path.join(path, "yolo_default.json")):
+                urlretrieve(file_url, os.path.join(path, "yolo_default.json"))
+                if verbose:
+                    print("Downloaded metadata json.")
+            else:
+                if verbose:
+                    print("Metadata json file already exists.")
```

Comment:
This could be simplified (same for the other instances below)
```suggestion
            elif verbose:
                print("Metadata json file already exists.")
```